src/include/distributed/version_compat.h

```diff
@@ -40,6 +40,7 @@
 #define standard_planner_compat(a,b,c,d) standard_planner(a,b,c,d)
 #define PortalDefineQuerySelectCompat(a,b,c,e,f) PortalDefineQuery(a,b,c,CMDTAG_SELECT,e,f)
 #define getOwnedSequencesCompat(a,b) getOwnedSequences(a)
+#define ExplainOnePlanCompat(a,b,c,d,e,f,g,h) ExplainOnePlan(a,b,c,d,e,f,g,h)
```

Comment:
Instead of `h` the pg13 version of this macro should have `NULL` be specified by the macro

So `#define ExplainOnePlanCompat(a,b,c,d,e,f,g) ExplainOnePlan(a,b,c,d,e,f,g,NULL)`